Console/retro/ConsoleWindow.cc

```diff
@@ -36,7 +37,9 @@ ConsoleWindow::ConsoleWindow(Rect r, ConstStr255Param title)
 {
     GrafPtr port;
         //Retro68 Improved Console
-    win = NewWindow(NULL, &r, "\pRetro68 Console", true, 0, (WindowPtr)-1, true, 0);
+    win = NewWindow(NULL, &r, "\p", true, 0, (WindowPtr)-1, true, 0);
+    setWindowName("Retro68 Console");
```

Comment:
It probably causes some minor flicker to use `setWindowName` instead of the parameter here

Reply:
I agree. Once the NewWindow is called with the right title, the next call to setWindowName is not needed, apart that it leaves the `windowName` private parameter undefined.

Reply:
I think the windowName member should only be used while parsing an escape sequence (it won't be undefined, it'll be default-constructed to empty). That was also the idea behind my `std::move` suggestion.

Reply:
Yes, I agree, I was thinking about it and I reached the same conclusion when I studied why `std::move` was relevant.